src/app/static/package.json

```diff
@@ -9,7 +9,7 @@
     "build": "rm -rf public && vue-cli-service build ./src/app/index.ts",
     "serve": "rm -rf public && VUE_CLI_SERVICE_CONFIG_PATH=$PWD/vue.config.js vue-cli-service serve ./src/app/index.ts",
     "copy": "cp -r ./templates/*.ftl ./public && cp -r ./assets/* ./public",
-    "test": "vitest --run",
+    "test": "vitest",
```

Comment:
This runs tests with watch?

Reply:
yh was just for while i was fiddling around with tests and debugging, this is reverted to `--run` later. But perhaps i should also have a "test:dev" script so that it is easier for us? Can do that in a future PR